packages/batch-delegate/src/createBatchDelegateFn.ts

```diff
@@ -20,7 +21,7 @@ export function createBatchDelegateFn<K = any, V = any, C = K>(
         args: argFn(keys),
         ...batchDelegateOptionsFn(options),
       });
-      return Array.isArray(results) ? results : keys.map(() => results);
```

Comment:
Results may be an Error and not an array, in which case, the error is currently sent to each key. Should this behavior be preserved?

Reply:
Ahh that makes sense. I added it back in